docs/source/getting_started/tutorials/analytical_machine_opt_tutorial/index.rst

```diff
@@ -104,9 +104,9 @@ The implementation of the ``ProblemDefinition`` protocol is provided in the foll
 Step 4.2: Analyzer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-As mentioned in the previous sub-step, the ``Analyzer`` protocol is designed to allow for modular and generalized analysis process to be utilized by multiple optimization. Typically implementations of ``Analyzers`` should be code which is lengthy enough to warrant the added complexity over the ``EvaluationStep`` protocol, or be code which is designed to be utilized by multiple optimization. For this example, the ``Analyzer`` is simple and could have been defined as an ``EvaluationStep`` but is defined this way to demonstrate how to implement the relevant classes. 
+As mentioned in the previous sub-step, the ``Analyzer`` protocol is designed to allow for modular, generalized, and reusable analysis code that is capable of studying aspects of multiple machine types. It usually makes sense to create an ``Analyzer`` for code that is lengthy enough to warrant the added complexity over the ``EvaluationStep`` protocol and/or is likely to be useful to other developers to study multiple machine types. `mach_cad` comes with many `Analyzer` classes already provided (browse the `Analyzers` section of the table of contents). Developers are encouraged to contribute their analyzers to this collection for the benefit of others.
```

Comment:
@elsevers I think it has to be `mach_eval` instead of `mach_cad` in the third sentence here.

Reply:
Good catch!